internal/auth/yaml_storage.go

```diff
@@ -20,6 +22,13 @@ type YamlStorage struct {
 func (y *YamlStorage) Setup(config config.AuthStorage, encryptionPassphrase string) error {
 	y.StorageEncryptionKey = encryptionPassphrase
 	y.StoragePath = config.Config["path"].(string)
+	if strings.HasPrefix(y.StoragePath, "~/") {
```

Comment:
This should check for `"~"+filepath.Separator`, otherwise it won't work properly on Windows.